docker/test/stateless/Dockerfile

```diff
@@ -52,6 +53,12 @@ RUN mkdir -p /tmp/clickhouse-odbc-tmp \
    && odbcinst -i -s -l -f /tmp/clickhouse-odbc-tmp/share/doc/clickhouse-odbc/config/odbc.ini.sample \
    && rm -rf /tmp/clickhouse-odbc-tmp
 
+ARG TARGETPLATFORM
+RUN if [ "$TARGETPLATFORM" = "linux/arm64" ]; then ARCHITECTURE=aarch64; else ARCHITECTURE=amd64; fi \
```

Comment:
Does DuckDB support PowerPC and RISC-V?

Reply:
I found no official information about it, only one [issue](https://github.com/duckdb/duckdb/issues/1998) with PowerPC and DuckDB v0.2.7